test/datasets/cmuarctic_test.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+from torchaudio.datasets import cmuarctic
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+class TestCMUARCTIC(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    samples = []
+
+    @classmethod
+    def setUpClass(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        sample_rate = 16000
+        seed = 42
+        utterance = "This is a test utterance."
+
+        base_dir = os.path.join(cls.root_dir, "ARCTIC", "cmu_us_aew_arctic")
+        txt_dir = os.path.join(base_dir, "etc")
+        os.makedirs(txt_dir, exist_ok=True)
+        txt_file = os.path.join(txt_dir, "txt.done.data")
+        audio_dir = os.path.join(base_dir, "wav")
+        os.makedirs(audio_dir, exist_ok=True)
+
+        with open(txt_file, "w") as txt:
+            for c in ["a", "b"]:
+                for i in range(5):
+                    utterance_id = f"arctic_{c}{i:04d}"
+                    path = os.path.join(audio_dir, f"{utterance_id}.wav")
+                    data = get_whitenoise(
+                        sample_rate=sample_rate,
+                        duration=3,
+                        n_channels=1,
+                        dtype="int16",
+                        seed=seed,
```

Comment:
Please use different `seed` value for each generated samples, otherwise all the loaded Tensors have the exactly same shape and value, and that will lose the point of comparing loaded Tensor object.